lib/node_modules/@stdlib/blas/ext/base/sapxsumkbn2/benchmark/benchmark.js

```diff
@@ -39,33 +43,27 @@ var sapxsumkbn2 = require( './../lib/sapxsumkbn2.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = new Float32Array( len );
-	for ( i = 0; i < x.length; i++ ) {
-		x[ i ] = ( randu()*20.0 ) - 10.0;
-	}
-	return benchmark;
-
-	function benchmark( b ) {
-		var v;
-		var i;
-
-		b.tic();
-		for ( i = 0; i < b.iterations; i++ ) {
-			v = sapxsumkbn2( x.length, 5.0, x, 1 );
-			if ( isnan( v ) ) {
-				b.fail( 'should not return NaN' );
-			}
-		}
-		b.toc();
-		if ( isnan( v ) ) {
-			b.fail( 'should not return NaN' );
-		}
-		b.pass( 'benchmark finished' );
-		b.end();
-	}
+    var x = filledarrayBy( len, 'float32', rand );
```

Comment:
The indentation changes will have to be reverted. Should be tab-indentation in JS files.